### DISCUSSION TITLE
Where is config file for theme

### DISCUSSION THREAD

**Discussion by jigapate** [rappasoft/laravel-livewire-tables#1670] [General]
Hello,

Where can i find config file to change default theme to bootstrap?
Also, can i apply my custom bootstrap theme which i use for my website to table?

Thank you for help,

**Comment by lrljoe** [comment]
You'd need to publish the config file to set the default theme to bootstrap:
```
php artisan vendor:publish --tag='livewire-tables-config'
```
Which will add a "config/livewire-tables.php" file

As for custom bootstrap themes, it depends what you mean by that!

You have the setTrAttributes() and setTdAttributes() to add in additional classes to rows/columns (or replace with the default parameter)
